gym-game2048/gym_game2048/main.py

```diff
@@ -0,0 +1,27 @@
+from gym_game2048.envs.env import Game2048Env
+from gym_game2048.envs.game_2048 import Game2048
+import numpy as np
+
+def main():
```

Comment:
Main driver of the program which creates an environment and takes random actions until the game ends. At each step, the action, reward, score and next state are printed to the terminal